basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function calculateDaysUntilToNextChristmas() {
```

Comment:
We had a discussion today, that this function should accept date from which we calculate number of days